CONTRIBUTING.rst

```diff
@@ -3,15 +3,44 @@ Contributing to PySCIPOpt
 
 Code contributions are very welcome and should comply to a few rules:
 
-1. Compatibility with both Python-2 and Python-3
+0. Read PySCIPOpt design principles. (see further)
 
-#. All tests defined in the Continuous Integration setup need to pass:
+1. Compatibility with both Python-2 and Python-3.
 
-    - `.travis.yml <.travis.yml>`__
-    - `appveyor.yml <appveyor.yml>`__
+2. All tests defined in the Continuous Integration setup need to pass:
 
-#. New features should be covered by the tests, so please extend `tests <tests>`__
+    - `.travis.yml <.travis.yml>`__;
+    - `appveyor.yml <appveyor.yml>`__.
```

Comment:
Why the trailing comma/semicolon?

Reply:
I removed them. Someone likes ending items of bullet lists with semicolons and commas, and the very last item with a period.